editor/tests/org.fusesource.ide.projecttemplates.tests.integration/src/main/java/org/fusesource/ide/projecttemplates/tests/integration/wizards/FuseIntegrationProjectCreatorRunnableIT.java

```diff
@@ -159,6 +160,9 @@ public void tearDown() throws CoreException, InterruptedException, IOException {
 		
 		CommonTestUtils.closeAllEditors();
 		new StagingRepositoriesPreferenceInitializer().setStagingRepositoriesEnablement(false);
+		
+		String projectName = project != null ? project.getName() : String.format("%s-%s", getClass().getSimpleName(), camelVersion);
+		ScreenshotUtil.saveScreenshotToFile(String.format("%s/MavenLaunchOutput-%s_AFTER.png", SCREENSHOT_FOLDER, projectName), SWT.IMAGE_PNG);
```

Comment:
but we are after the teardown and all editors were closed.
The purpose to take the screenshot is to have a view of the state when the test has failed.
i'm fine to also see what's left after the teardown but in this case we need 2 screenshots